app/org/sagebionetworks/bridge/play/controllers/StudyController.java

```diff
@@ -125,6 +127,18 @@ public Result createStudy() throws Exception {
         return okResult(new VersionHolder(study.getVersion()));
     }
 
+    public Result createSynapse(String userId) throws Exception {
+        // first get current study
+        UserSession session = getAuthenticatedSession(DEVELOPER, RESEARCHER, ADMIN);
```

Comment:
Developer is the only necessary role. Researcher is just for IRB-sensitive information, and we haven't given ADMIN permission to do all the developer things (elsewhere).